[2024-04-24T16:16:56.475+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-24T16:16:56.548+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-22T00:00:00+00:00 [queued]>
[2024-04-24T16:16:56.571+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-22T00:00:00+00:00 [queued]>
[2024-04-24T16:16:56.572+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-24T16:16:56.637+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-22 00:00:00+00:00
[2024-04-24T16:16:56.704+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'scheduled__2024-04-22T00:00:00+00:00', '--job-id', '242', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpn6hytq6y']
[2024-04-24T16:16:56.709+0000] {standard_task_runner.py:91} INFO - Job 242: Subtask get_files
[2024-04-24T16:16:56.712+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=159) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-24T16:16:56.716+0000] {standard_task_runner.py:63} INFO - Started process 232 to run task
[2024-04-24T16:16:56.865+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.get_files scheduled__2024-04-22T00:00:00+00:00 [running]> on host af114bcfa67a
[2024-04-24T16:16:57.293+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-22T00:00:00+00:00'
[2024-04-24T16:16:57.298+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-24T16:16:57.310+0000] {logging_mixin.py:188} INFO - 9 files found in /opt/***/dags/data
[2024-04-24T16:16:57.317+0000] {python.py:237} INFO - Done. Returned value was: ['/opt/***/dags/data/olist_orders_dataset.csv', '/opt/***/dags/data/olist_products_dataset.csv', '/opt/***/dags/data/olist_order_payments_dataset.csv', '/opt/***/dags/data/product_category_name_translation.csv', '/opt/***/dags/data/olist_sellers_dataset.csv', '/opt/***/dags/data/olist_order_reviews_dataset.csv', '/opt/***/dags/data/olist_order_items_dataset.csv', '/opt/***/dags/data/olist_customers_dataset.csv', '/opt/***/dags/data/olist_geolocation_dataset.csv']
[2024-04-24T16:16:57.321+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-24T16:16:57.403+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240422T000000, start_date=20240424T161656, end_date=20240424T161657
[2024-04-24T16:16:57.557+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-24T16:16:58.114+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-24T16:16:58.138+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
